src/utils/metadata.py

```diff
@@ -90,14 +90,22 @@ def build_metadata_from_setuptools_dict(metadata):
     # https://github.com/pypa/warehouse/blob/master/warehouse/templates/packaging/detail.html
     # https://github.com/pypa/virtualenv/blob/master/setup.cfg
     project_urls = metadata["project_urls"]
+    if "Source" in project_urls:
+        repo_url = project_urls["Source"]
+    else:
+        repo_url = project_urls["Repository"]
```

Comment:
does capitalization matter?
Can you also link the spec for this? (not sure it's this https://peps.python.org/pep-0621/#example ?)

Reply:
Caps is not mandatory in the PEP621 specs but Pypi takes the keys as is, so if it is not capitalized it will not be either on Pypi (see URLs of https://test.pypi.org/project/TestProjectABerchet/0.1.0/ vs https://test.pypi.org/project/TestProjectABerchet/0.1.1/). That's why I used caps here.

Reply:
Thanks for the test; that's wild.